internal/usage/usage.go

```diff
@@ -36,6 +36,12 @@ const (
 	DBUsername                      = "Username for authenticating to MongoDB."
 	TeamName                        = "Name of the team."
 	UserID                          = "The ID of the user."
+	Hostname                        = "The hostname or IP address of the LDAP server."
+	Port                            = "The port to which the LDAP server listens for client connections. Default: 636"
```

Comment:
[non-blocking] we should name this `LDAPPort` to avoid someone copy/pasting the flag without realising the copy is ldap specific

Reply:
I am going to change them in the next PR